aten/src/ATen/mps/MPSDevice.h

```diff
@@ -66,12 +66,13 @@ class TORCH_API MPSDevice {
   static MPSDevice* _device;
   MTLDevice_t _mtl_device;
   bool _macos13plus;
+  bool _macos_13_1_plus;
```

Comment:
Lets not keep extra booleans here, since we just need to know which subversion number...

Reply:
Thanks. I've updated